src/fiskaltrust.Middleware.Interface.Client.Common/Helpers/ModelConverters.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Linq;
+
+namespace fiskaltrust.Middleware.Interface.Client.Helpers.ModelConverters
+{
+    static class Helper
+    {
+        public static T[] ConvertArray<U, T>(U[] from, Func<U, T> converter)
+        {
+            if(from == null) return null;
+
+            return from.Select(item => converter(item)).ToArray();
+        }
+    }
+    public static class v0

```

Comment:
I'd go with extension methods for this approach, something like this:
```cs
public static class ReceiptRequestExtensions {
  public static ifPOS.v0.ReceiptRequest FromV1(this ifPOS.v1.ReceiptRequest from) {...}
}
```
And call it like that:
```cs
var reqV0 = reqV1.FromV1();
```

The reason is that this very much looks like a constructor when calling it, which it isn't :)

Reply:
implemented `.Into()`